designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/StatisticsDb.scala

```diff
@@ -0,0 +1,24 @@
+package pl.touk.nussknacker.ui.db.timeseries
+
+import cats.effect.{IO, Resource}
+
+import scala.concurrent.Future
+import scala.language.higherKinds
+
+trait StatisticsDb[F[_]] extends AutoCloseable {
```

Comment:
1. the fact that the DB should be closed is also specific to a given implementation. If we expose the instance using a factory method that returns `Resource`, there is no need to declare it as `AutoCloseable`

2. I wonder if we shouldn't have `StatisticsRepository` interface, instead of the `StatisticsDb`?